enzyme/test/Enzyme/ForwardMode/remainder.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -O3 -S | FileCheck %s
```

Comment:
Can you change this to only do mem2reg simplifycfg and instsimplify?

We should check that the differeturn is handled properly

Reply:
Done, output looks correct to me.